cluster.go

```diff
@@ -18,19 +18,21 @@ import (
 // ErrNoSlot indicates that there is no redis node owns the key slot.
 var ErrNoSlot = errors.New("the slot has no redis node")
 var ErrReplicaOnlyConflict = errors.New("ReplicaOnly conflicts with SendToReplicas option")
+var ErrInvalidScanInterval = errors.New("scan interval must be greater than or equal to 0")
 
 type clusterClient struct {
-	pslots [16384]conn
-	rslots []conn
-	opt    *ClientOption
-	rOpt   *ClientOption
-	conns  map[string]connrole
-	connFn connFn
-	sc     call
-	mu     sync.RWMutex
-	stop   uint32
-	cmd    Builder
-	retry  bool
+	pslots      [16384]conn
+	rslots      []conn
+	opt         *ClientOption
+	rOpt        *ClientOption
+	conns       map[string]connrole
+	connFn      connFn
+	sc          call
+	mu          sync.RWMutex
+	stop        uint32
+	cmd         Builder
+	retry       bool
+	refreshStop chan struct{}
```

Comment:
```suggestion
	stopCh chan struct{}
```
The channel can be used by other goroutines in the future. Let's not limit its naming to the current purpose.

Reply:
[1181ffb](https://github.com/redis/rueidis/pull/638/commits/1181ffbe8041a47911e438c99bb2464e46a6d00e)